src/css/style.components.css

```diff
@@ -99,7 +99,50 @@
     text-decoration: none;
 }
 
-/* njk/md parser intoduces a bunch of stray <p> tags that we can do nothing about */
+/*
+    Table of Contents - Blog Posts
+*/
+.toc-wrapper .toc {
```

Comment:
@Yndira-E This is part AI, and part my doing. It's backend engineer quality. Can you take a look?